application/cs/ajax.texy

```diff
@@ -10,7 +10,24 @@ Moderní webové aplikace dnes běží napůl na serveru, napůl v prohlížeči
 
 </div>
 
-AJAXový požadavek lze detekovat metodou služby [zapouzdřující HTTP požadavek |http:request] `$httpRequest->isAjax()` (detekuje podle HTTP hlavičky `X-Requested-With`). Uvnitř presenteru je k dispozici "zkratka" v podobě metody `$this->isAjax()`.
+Ajaxový požadavek
+=================
+
+V prohlíčeči vytvoření ajaxového http požadavku zajistí javascriptová knihovna Naja. Aby se z obyčejného http požadavku (signál odkazu, odeslání formuláře, ...) vytvořil ajaxový požadavek, je potřeba označit příslušný odkaz (formulář, odesílací tlačítko formuláře):
```

Comment:
Odkaz na Naja která je na stránce odkázaná o kousek níž?

Reply:
Já sám jsem se zde na chvilku zaseknul, protože jsem se nedočetl jak se ajaxový požadavek vytvoří, ale napadlo mě si to v odkazu na Naja najít. Syn, který byl na školení Nette a znal odtamtud <a href=".." class="ajax"> ale nevěděl jak odeslat ajaxově formulář, hledal to v dokumentaci, ale nenapadlo ho jít do odkazu Naja. Sám na to nepřišel, podobně i druhý syn se také zasekával na podobných drobnostech.

Přijde mě logické, když se začne mluvit o zpracování ajaxového požadavku dát před to dvě-tři věty jak ajaxový požadavek vznikne. Aby v odstavci v úvodu byly tyto informace pohromadě.

Druhou možností je přesunout vzorek kódu který jsem poslal do sekce o knihovně Naja.

Reply:
Já tím myslel, že by to chtělo prolinkovat na kapitolu o Naja. Důvod chápu.